CHANGELOG.md

```diff
@@ -35,31 +34,40 @@ For example:
 @import "node_modules/govuk-frontend/govuk/components/header";
 ```
 
+Find out more about [importing CSS](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#css).
+
 [Pull request #1804: Allow components to be imported without dependencies](https://github.com/alphagov/govuk-frontend/pull/1804).
 
+#### Back links and breadcrumbs are easier to select
```

Comment:
Select doesn't sound quite right to me – does 'activate' work? That's the term that's used in the [related WCAG success criteria](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html).

Reply:
It looks like 'select' is GOV.UK preferred term: https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#click. (And it does feel plainer language to me than 'activate'.)

Reply:
👍